tests/UnitElab/UnitElab.log

```diff
@@ -17362,6 +17427,117 @@ design: (work@bottom1)
         \_logic_net: (work@top.inst1.c), line:20:25, endln:20:26
       |vpiTypedef:
       \_logic_typespec: , line:22:8, endln:22:8
+  |vpiModuleArray:
```

Comment:
This is redundant information

Reply:
For Generate statements we do not keep any folded view, we bit blast, same with the gate arrays, we have to be consistent. You can keep the module_arrays and instance_arrays in the non-elaborated model.

Reply:
Plus tools like Yosys and Verilator would see 2 views of the same thing, errors in perspective.
This PR cannot go in as is